### PR TITLE
Elements: Scope style rules to element template only

### PR DESCRIPTION
**Pull request by maurofmferrao**
relates to xibosignageltd/xibo-private#735

### CODE REVIEW

File: modules/src/xibo-player.js [hunk 1]
```diff
@@ -704,6 +704,28 @@ XiboPlayer.prototype.init = function() {
   // Create global render array of functions
   window.renders = [];
 
+  // If we have scoped styles
+  // convert the CSS rules to use it
+  $('[data-style-scope]').each((_idx, styleEl) => {
```
Comment (dasgarner) [inline modules/src/xibo-player.js:709]: Does this handle a widget cached before this change? I.e. where there is no type/datatype/templateId ?

File: modules/widget-html-render.twig [hunk 2]
```diff
@@ -50,11 +50,17 @@
...
         </style>
         {% if style|length > 0 %}
-        <style type="text/css">
             {% for item in style %}
-            {{ item|raw }}
+                {% if item.content %}
+                    <style type="text/css" data-style-scope="{{item.type}}_{{item.dataType}}__{{item.templateId}}">
+                        {{ item.content|raw }}
+                    </style>
+                {% else %}
```
Comment (dasgarner) [inline modules/widget-html-render.twig:58]: This should never be the case because the WidgetHtmlRenderer has been changed at the same time.

**Review by rubenberttpingol — APPROVED** [review]

**Review by dasgarner — APPROVED** [review]

**Comment by ifarzana** [comment]
TEST: /Layout END

**Comment by ifarzana** [comment]
TEST: /Layout/Editor/layout_editor_mastodon.cy.js END

**Pull request merged**